src/content_delegate.js

```diff
@@ -482,14 +482,11 @@ ContentDelegate.prototype.showPdfPage = function(
               }
             }.bind(this);
 
-            // In Chrome, blobs can't be passed from content scripts
-            // to background scripts, so we have to convert to an
-            // array and pass that, then convert back to a blob when
-            // we add the data to the FormData object.
-            let bytes = arrayBufferToArray(ab);
```

Comment:
Can you also remove the arrayBufferToArray function? I think it's no longer used.

Reply:
Done.